packages/core/src/common/os.ts

```diff
@@ -65,7 +67,8 @@ export namespace OS {
     export const backend = {
         type,
         isWindows,
-        isOSX
+        isOSX,
+        EOL
```

Comment:
This property needs to be overriden [here](https://github.com/eclipse-theia/theia/blob/fbb02df763b115c12e2d68babf33aac3a781b449/packages/core/src/browser/preloader.ts#L50-L58) as well. Otherwise it will stay as the frontend EOL, which isn't the actual backend EOL.